examples/containers/multi/run.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#####
+
+# Pre-conditions: 
+#   - No containers named "cmos" or "node$i" where $i is an integer up to the number
+#     of nodes desired in the cluster ($NUM_NODES). Checked by the script and handled by 
+#     the user, either via destroying them manually (e.g., make clean) or agreeing to them
+#     being removed by the script.
+#   - The couchbase/observability-stack Docker image built (handled by the Makefile)
+
+# Post-conditions: 
+#   - A single container named "cmos" with the CMOS Microlith running. 
+#   - A total of $NUM_NODES containers with the specified Couchbase Server version
+#     and Prometheus exporter installed, partitioned as evenly as possible into $NUM_CLUSTERS
+#   - cbmultimanager configured to monitor all clusters
+#   - Grafana configured to retrieve statistics from CBMM API for dashboards
+set -eu
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+DOCKER_USER=${DOCKER_USER:-couchbase}
+DOCKER_TAG=${DOCKER_TAG:-v1}
+CMOS_IMAGE=${CMOS_IMAGE:-$DOCKER_USER/observability-stack:$DOCKER_TAG}
+
+# Disable check as checked elsewhere
+# shellcheck disable=SC1091
+source "$SCRIPT_DIR"/helpers/driver.sh
+
+# Environment variables
```

Comment:
That's nice but not very helpful, we should say what the variables are for, some are a big vague, e.g. `DEV`